instrumentation/spring/spring-webmvc-3.1/library/README.md

```diff
@@ -79,12 +79,12 @@ import org.springframework.web.client.RestTemplate;
 public class WebMvcTracingFilterConfig {
 
    @Bean
-   public WebMvcTracingFilter webMvcTracingFilter(Tracer tracer) {
-      return new WebMvcTracingFilter(tracer);
+   public WebMvcTracingFilter webMvcTracingFilter(OpenTelemetry openTelemetry) {
```

Comment:
```suggestion
   public Filter webMvcTracingFilter(OpenTelemetry openTelemetry) {
```

Reply:
I believe that out of the box this construct works only for spring boot, for plain spring mvc you'll need to also add the filter to web.xml. There is a sample in instrumentation/spring/spring-webmvc-3.1/wildfly-testing/src/test/resources/web.xml

Reply:
Yeah, that may be true - the whole instrumentations seems to be written with spring boot (and our spring-boot-autoconfigure module) in mind.
I'll add it to the README file.